astronomy/date_time_body.hpp

```diff
@@ -1038,6 +1091,10 @@ constexpr DateTime operator""_DateTime(char const* const str,
                                     size - (index_of_T + 1)));
 }
 
+inline std::ostream& operator<<(std::ostream& out, DateTime const& date_time) {
+  return out << date_time.date() << "T" << date_time.time();
```

Comment:
Where is this operator (and the previous ones) tested.?

Reply:
In #3040.